src/platform/packages/shared/kbn-unified-metrics-grid/src/components/metrics_experience_grid.tsx

```diff
@@ -45,8 +45,8 @@ export const MetricsExperienceGrid = ({
   const { euiTheme } = euiThemeContext;
 
   const { currentPage, dimensions, valueFilters, onPageChange } = useMetricsGridState();
-
   const { getTimeRange, updateTimeRange } = requestParams;
+  const [debouncedSearchTerm, setDebouncedSearchTerm] = useState('');
```

Comment:
nit: this component doesn't know whether the set function will be debounced

```suggestion
  const [searchTerm, setSearchTerm] = useState('');
```

Also, shouldn't we use the redux state instead?

Reply:
Answered [here](https://github.com/elastic/kibana/pull/234299#discussion_r2353060140) the debounce is removed